### PR TITLE
fixed all warnings in reducers/map.js

### PR DESCRIPTION
**Pull request by ochanje210**

### CODE REVIEW

File: src/shared/reducers/map.js [hunk 1]
```diff
@@ -2,28 +2,28 @@ import * as types from '../actions/ActionTypes';
...
-		error: -1
-	}
+  get: {
+    status: 'INIT',
+    markers: [],
+    error: -1
+  }
 };
 
 export default function map(state, action) {
```
Comment (fguy) [inline src/shared/reducers/map.js:12]: Please use arrow function:

`export default (state, action) => {...};`

File: src/shared/components/MapBlock.js [hunk 2]
```diff
@@ -5,84 +5,86 @@ import controllable from 'react-controllables';
...
+    this.onBoundsChange = (center, zoom, bounds, marginBounds) => {
+      this.props.onCenterChange(center);
+      this.props.onZoomChange(zoom);
+    };
+
+    this.onChildClick = (key, childProps) => {
+      this.props.onCenterChange([childProps.lat, childProps.lng]);
+    };
+
+    this.onChildMouseEnter = (key, childProps) => {
```
Comment (fguy) [inline src/shared/components/MapBlock.js:23]: Do we have to determine these empty methods?

File: src/shared/components/MapBlock.js [hunk 3]
```diff
@@ -5,84 +5,86 @@ import controllable from 'react-controllables';
...
-			</section>
-		);
-	}
+class MapBlock extends Component {
+  constructor(props) {
+    super(props);
 
+    this.shouldComponentUpdate = shouldPureComponentUpdate;
+
+    this.onBoundsChange = (center, zoom, bounds, marginBounds) => {
```
Comment (fguy) [inline src/shared/components/MapBlock.js:14]: You can extract this.XXX as members of MapBlock class like below:

`
class MapBlock extends Component {
  onBoundsChange = (...) => {}
  onChildClick = (...) => {}
...
}
`

File: src/shared/components/MapBlock.js [hunk 4]
```diff
@@ -5,84 +5,86 @@ import controllable from 'react-controllables';
...
-					onChildClick={this._onChildClick}
-					onChildMouseEnter={this._onChildMouseEnter}
-					onChildMouseLeave={this._onChildMouseLeave}
-					hoverDistance={20}>
-					{markers}
-				</GoogleMap>
-			</section>
-		);
-	}
+class MapBlock extends Component {
```
Comment (fguy) [inline src/shared/components/MapBlock.js:8]: You can merge export line and class definition like this:

`export default class extends Component {...}`

File: src/shared/containers/MapTest.js [hunk 5]
```diff
@@ -5,30 +5,32 @@ import controllable from 'react-controllables';
 import GoogleMap from 'google-map-react';
 import Marker from '../components/Marker.js';
 
-class MapTest extends Component {  
-
-  
+class MapTest extends Component {
```
Comment (fguy) [inline src/shared/containers/MapTest.js:8]: What's the purpose of this test class? Please use enzyme for react component test. http://airbnb.io/enzyme/

**Comment by fguy** [comment]
Looks good overall. Please see inline comments.

File: src/shared/reducers/map.js (hunk 1 above)
Comment (ochanje210) [reply src/shared/reducers/map.js:12]: ok, I will follow that now

**Comment by fguy** [comment]
Wait. Where is test codes? Please make sure all your change comes with tests.

File: src/shared/components/MapBlock.js [hunk 6]
```diff
@@ -5,84 +5,86 @@ import controllable from 'react-controllables';
...
-    	this.shouldComponentUpdate = shouldPureComponentUpdate;
-
-    	this._onBoundsChange = (center, zoom, bounds, marginBounds) => {  		
-		    this.props.onCenterChange(center);
-		    this.props.onZoomChange(zoom);	    
-	  	}
-
-	  	this._onChildClick = (key, childProps) => {
-	  		this.props.onCenterChange([childProps.lat, childProps.lng]);
-		}
```
Comment (shberry16) [inline src/shared/components/MapBlock.js:22]: Indentation is getting a bit weird from here. Can you take a look? :)

File: src/shared/reducers/map.js [hunk 7]
```diff
@@ -2,28 +2,28 @@ import * as types from '../actions/ActionTypes';
...
+    return update(thisState, {
+      get: {
+        status: { $set: 'SUCCESS'},
+        markers: { $set: action.data }
+      }
+    });
+  default:
+    return thisState;
+  }
+}
```
Comment (shberry16) [inline src/shared/reducers/map.js:29]: Please remove extra line at the end of the file if it's not necessary :)

**Pull request merged**